testes/word-rank/resposta/wordrank.js

```diff
@@ -0,0 +1,55 @@
+var fs = require('fs');
+var buf = fs.readFileSync(process.argv[2]);
+var result = buf.toString();
+var rank = [];
+
+//Função que encontra o index do array do elemento 
+function findArray(element) {
+  var indexFound = rank.findIndex(function(minAr) {
+    return minAr[0] === element;
+  });
+
+  return indexFound;
+}
+
+//Função que ordena o array por ordem alfabetica e por numero de ocorrencias da palavra
+function sortArray(a, b) {
+  if(a[0] < b[0] && a.length >= b.length) {
+    return -1;
+  }
+  if(a[0] < b[0] && a.length < b.length) {
+    return 1;
+  }
+  if(a[0] > b[0] && a.length > b.length) {
+    return -1;
+  }
+  if(a[0] > b[0] && a.length <= b.length) {
+    return 1;
+  }
```

Comment:
@medson10 - Revisar essa parte da ordenação com o que é pedido no exercício, você precisa ordenar por frequência primeiro e se a frequência for igual, ordenar alfabeticamente.